### PR TITLE
fix options when host is set

### PR DESCRIPTION
**Pull request by sfc-gh-bculberson**
### Description
The host configuration which is in the list of options is not used to construct the accessUrl when it should be.

### Checklist
- [x] Format code according to the existing code style (run `npm run lint:check -- CHANGED_FILES` and fix problems in changed code)
- [x] Create tests which fail without the change (if possible)
- [x] Make all tests (unit and integration) pass (`npm run test:unit` and `npm run test:integration`)
- [x] Extend the README / documentation and ensure is properly displayed (if necessary)
- [x] Provide JIRA issue id (if possible) or GitHub issue id in commit message

### CODE REVIEW

**Comment by sfc-gh-bculberson** [comment]
I have read the CLA Document and I hereby sign the CLA

File: lib/connection/connection_config.js [hunk 1]
```diff
@@ -81,7 +81,9 @@ function consolidateHostAndAccount(options) {
     if (options.region === 'us-west-2') {
       options.region = '';
     }
-    if (dotPos < 0 && Util.isString(options.region) && options.region.length > 0) {
+    if (Util.exists(options.host)) {
```
Comment (sfc-gh-pbulawa) [inline lib/connection/connection_config.js:84]: Could you please add some test cases for that (file `test/unit/connection/connection_config_test.js`).

**Review by sfc-gh-pbulawa — CHANGES_REQUESTED** [review]

File: lib/connection/connection_config.js (hunk 1 above)
Comment (bculberson) [reply lib/connection/connection_config.js:84]: i am unable to run tests

**Comment by sfc-gh-pmotacki** [comment]
Hello @sfc-gh-bculberson. This change was merged by @sfc-gh-pbulawa in PR #749. Thank you for your commitment. I will be released in next version

**Pull request closed without merging**